### ISSUE TITLE
Get consent on publicly linking name & CSTK holder address

### ISSUE THREAD

**Issue by lorelei1** [commons-stack/iteration0#88]
## 🎉   Subtasks

- [x] Grab the 200 names, put them in a spreadsheet with contact.
- [x] Write up the sample message(s).
- [x] Have the message(s) approved.
- [x] Create campaign(s) on Mailchimp.
- [x] Send emails.
- [ ] Update spreadsheet with consent and current/change of address (Ongoing)
- [x] Draft the second email.
- [x] Have the adjustments for the second email approved.
- [x] Draft Telegram messaging.
- [x] Have Telegram messaging approved.
- [x] Send a second email!
- [x] Execute Telegram outreach.
- [x] Team members assign their names next to a person to contact
- [ ] DM everybody, update their consent with current address or change of address
- [ ] Review

## 🤼‍♂️   Reviewer

@GriffGreen 

## 🔗   Work doc - inspirational links
Look for "CSTK holders to contact for consent" in our shared drive

**Comment by lorelei1** [comment]
@GriffGreen can you remind me did we figure out a certain date that we needed this done by in order for the roadmap to progress? 
I know that we won't mint their xDai CSTK until they have consented... but are there any external factors? The panvala draft?

**Comment by GriffGreen** [comment]
Yeah the Panvala draft says we should get things done by September 8th!

**Comment by lorelei1** [comment]
Sample message: 
"Hey _________! Are you cool with having your name and the ETH address you gave us being linked publicly? 
We're going to do Praise a bit more transparently in the future, and you applied before we actually put that consent clause in the typeform. So I want to get your confirmation first :)"

**Comment by lorelei1** [comment]
Sounds like we need to decide on whether we are keeping CSTK holders under 3k anonymous (and lifting the burden on DMing all of them)

**Comment by GriffGreen** [comment]
I like having some token limit that allows people to be anon (but still people) 3000 seems cool :-D

**Comment by GriffGreen** [comment]
My sample message:

Hey ____Person's name_____! Are you cool with having your name linked to this address: _____0xblahblahblah_____

We made a mistake not asking for a publicly linkable address early on when we made the application and didn't want to dox people with our Praise system so we have just been doing it behind the scenes...

But we are moving to xDAI and its the perfect time to make the Praise system open so people can see what earns CSTK tokens AND show the TE Commons who holds CSTK tokens so they can feel comfortable letting CSTK Token holders participate in the Hatch. 

So is that address ok or would you prefer a different publicly linkable address to hold your CSTK Tokens on xDAI?

**Comment by GriffGreen** [comment]
made some progress on this last week :-D checked all the addys and checked in with the high risk whales and got some new addresses

**Comment by GriffGreen** [comment]
Ask @lorelei1 about the transparent burn log!!

**Comment by GriffGreen** [comment]
Hey **Person's name_**! 

Are you cool with having your name publicly known as part of the Trusted Seed and linked to this address: **0xblahblahblah**?

We are preparing to launch the TE Commons and as an accepted member of the Trusted Seed, if you have CSTK tokens, you will be able to participate in the TEC launch! While we strive to keep your address private, we will be listing the members of the Trusted Seed, and through the Praise process, there is a chance that someone could link your name to the address that holds your CSTK/TEC Tokens. 

We made a mistake not asking for a public Ethereum address when we first started collecting applications and want to give you the option to give us a new address. If you do want to give us a new address, ideally it will be easy to use with DApps on xDAI. We suggest a new account on Metamask seed that you have already backed up well.

Thank you for your quick reply,
